tiny86/test/codegen/e2egen

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/nix/store/wcy993wz0zlcixaxivi6gha33afwq66n-ruby-3.1.4/bin/ruby
```

Comment:
Does nix's `patchShebangs` not handle this correctly for some reason?

Reply:
When I built in a fresh VM, using the nix store path resulted in the script not being able to find ruby. Reverting to what was originally there fixes it.